webapp/channels/src/plugins/rhs_plugin/rhs_plugin.tsx

```diff
@@ -13,26 +13,24 @@ export type Props = {
     title: React.ReactNode;
 }
 
-export default class RhsPlugin extends React.PureComponent<Props> {
-    render() {
-        return (
-            <div
-                id='rhsContainer'
-                className='sidebar-right__body'
-            >
-                <SearchResultsHeader>
-                    {this.props.title}
-                </SearchResultsHeader>
-                {
-                    this.props.showPluggable &&
-                    <>
-                        <Pluggable
-                            pluggableName='RightHandSidebarComponent'
-                            pluggableId={this.props.pluggableId}
-                        />
-                    </>
-                }
-            </div>
-        );
-    }
-}
+const RhsPlugin = ({showPluggable, pluggableId, title}: Props) => {
+    return (
+        <div
+            id='rhsContainer'
+            className='sidebar-right__body'
+        >
+            <SearchResultsHeader>
+                {title}
+            </SearchResultsHeader>
+            {
+                showPluggable &&
+                <Pluggable
+                    pluggableName='RightHandSidebarComponent'
+                    pluggableId={pluggableId}
+                />
+            }
+        </div>
+    );
+};
+
+export default RhsPlugin;
```

Comment:
It was a `PureComponent`, so we need `React.memo` here to avoid re-renders.
```suggestion
export default React.memo(RhsPlugin);
```